pkg/venafi/firefly/connector.go

```diff
@@ -59,9 +60,22 @@ func (c *Connector) RetrieveAvailableSSHTemplates() (response []certificate.SshA
 
 // NewConnector creates a new Firefly Connector object used to communicate with Firefly
 func NewConnector(url string, zone string, verbose bool, trust *x509.CertPool) (*Connector, error) {
+	if url != "" {
+		var err error
```

Comment:
This declaration is not required.
below line can do
`url, err := normalizeUrl()`

Reply:
that is not correct, given that create a new `user` var which only lives in the scope of the surrounder `if condition`

Reply:
got it